### PR TITLE
applied copyright changes

### PR DESCRIPTION
**Pull request by Akyiaa**
Signed-off-by: Fiona Ampofo <64271621+Akyiaa@users.noreply.github.com>
Story: https://github.com/orgs/galasa-dev/projects/3/views/1?filterQuery=1527&pane=issue&itemId=33443516

### CODE REVIEW

**Comment by galasa-team** [comment]
Build successful

File: galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/internal/rps/tree/ASTConstant.java
Comment (Akyiaa) [inline galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/internal/rps/tree/ASTConstant.java:1]: should the copyright comment go after the comment already present here?

**Review by Wyvinar — APPROVED** [review]
It looks good! 
Interesting that in certain areas the "Copyright contributors to the Galasa project" is deleted and replaced and in others it isn't. or at least doesn't appear to. It works though!

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Review by techcobweb — APPROVED** [review]

File: galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/internal/rps/tree/ASTConstant.java
Comment (techcobweb) [reply galasa-parent/dev.galasa.framework/src/main/java/dev/galasa/framework/internal/rps/tree/ASTConstant.java:1]: I think it's OK as it is...

**Pull request merged**